fedimintd/templates/params.html

```diff
@@ -35,11 +35,11 @@ <h3>Configure Federation</h3>
     </div>
     <div class="form-group">
       <label for="guardians_count" class="col-form-label">Number of Guardians Including You</label>
-      <input class="form-control" type="number" name="guardians_count" required placeholder="4" />
+      <input class="form-control" type="number" name="guardians_count" required placeholder="4" min="1" oninput="this.value = Math.abs(this.value)" />
     </div>
     <div class="form-group">
       <label for="finality_delay" class="col-form-label">Finality delay</label>
-      <input class="form-control" type="number" name="finality_delay" required value="10" />
+      <input class="form-control" type="number" name="finality_delay" required value="10" min="1" oninput="this.value = Math.abs(this.value)" />
```

Comment:
why `min` and the js hack?

Reply:
So users can't input negative values, or 0. Seemed like the [least invasive way to accomplish this](https://stackoverflow.com/questions/7372067/is-there-any-way-to-prevent-input-type-number-getting-negative-values)

Reply:
But that's what the `min` attribute is there for already. I was wondering about the additional JS.